torch/_dynamo/eval_frame.py

```diff
@@ -311,6 +311,15 @@ def get_compiler_fn(compiler_fn):
 def lookup_backend(compiler_fn):
     """Expand backend strings to functions"""
     if compiler_fn == "inductor":
+        if (
+            torch.backends.cuda.matmul.allow_tf32 is False
+            and torch.cuda.get_device_capability() >= (8, 0)
+        ):
+            warnings.warn(
+                "Tensor cores are available but not enabled."
```

Comment:
TensorFloat32 tensor cores for float32 matrix multiplication available but not enabled. Consider setting torch.set_float32_matmul_precision('HIGH')